File: 2006Scape Server/src/main/java/com/rs2/integrations/discord/commands/Website.java
package com.rs2.integrations.discord.commands;

import com.rs2.Constants;
import org.javacord.api.entity.message.Message;
import org.javacord.api.event.message.MessageCreateEvent;
import org.javacord.api.listener.message.MessageCreateListener;

public class Website implements MessageCreateListener {
    @Override
    public void onMessageCreate(MessageCreateEvent event) {
        Message message = event.getMessage();
        if (message.getContent().equalsIgnoreCase("::website") || message.getContent().equalsIgnoreCase("::site")) {
            if (Constants.WORLD == 1) {
                event.getChannel().sendMessage(Constants.WEBSITE_LINK);
            }
        }
    }
}

File: 2006Scape Server/src/main/kotlin/com/rs2/integrations/discord/commands/Website.kt
package com.rs2.integrations.discord.commands

import com.rs2.GameConstants
import org.javacord.api.event.message.MessageCreateEvent
import org.javacord.api.listener.message.MessageCreateListener

class Website : MessageCreateListener {
    override fun onMessageCreate(event: MessageCreateEvent) {
        val message = event.message
        if (message.content.equals("::website", ignoreCase = true) || message.content.equals(
                "::site",
                ignoreCase = true
            )
        ) {
            if (GameConstants.WORLD == 1) {
                event.channel.sendMessage(GameConstants.WEBSITE_LINK)
            }
        }
    }
}